src/ui/common/src/components/DataPreviewer/index.tsx

```diff
@@ -1,169 +0,0 @@
-import Alert from '@mui/material/Alert';
```

Comment:
I take it this file is no longer used?

Reply:
Yes, I did CRTL + F DataPreviewer and I didn't see it used anywhere.

Reply:
Sounds good 👌